pkg/fdbadminclient/lock_client.go

```diff
@@ -3,7 +3,7 @@
  *
  * This source file is part of the FoundationDB open source project
  *
- * Copyright 2018-2021 Apple Inc. and the FoundationDB project authors
+ * Copyright 2018-2024 Apple Inc. and the FoundationDB project authors
```

Comment:
Are these updates coming from some kind of automated update script? If so, does that script need to be updated to reflect that, if I'm not mistaken, it's currently 2025?

Reply:
I started some changes in December last year and seems like I missed those. Right now we don't have any automated way to update the headers.